### DISCUSSION TITLE
How to make a script run at the beggining of opening a kitty instance?

### DISCUSSION THREAD

**Discussion by smartguy4321** [kovidgoyal/kitty#5791] [Q&A] [answered]
For example, If i wanted to have neofetch, or a color-script show at the start of every kitty instance, what would I need to put in kitty.conf? I have tried putting both `neofetch` and `kitty neofetch` and have had no results with either. How would this be done? Thanks in advance to anyone who answers!

**Comment by kovidgoyal** [comment, marked as answer]
On Sat, Dec 17, 2022 at 03:51:00PM -0800, smartguy4321 wrote:
> For example, If i wanted to have neofetch, or a color-script show at the start of every kitty instance, what would I need to put in kitty.conf? I have tried putting both `neofetch` and `kitty neofetch` and have had no results with either. How would this be done? Thanks in advance to anyone who answers!

shell "neofetch; bash -l -i"


replace bash with whatever shell you actuall yuse. But beware this will
break shell integration. Instead put neofetch in your shell's rc file.

**Reply by smartguy4321** [reply]
Thank you, this helped!
